example_usage.py

```diff
@@ -0,0 +1,122 @@
+"""
+Example of how to use fastapi-zitadel-auth testing utilities in your own project
+"""
```

Comment:
can we use a real-world example with multiple files?

e.g. 

a `main.py` for the router and dependencies
a `conftest.py`
a `test_stuff.py`

which all should be a Markdown file that we can link to in the MKdocs package later https://github.com/cleanenergyexchange/fastapi-zitadel-auth/blob/c4a11bc318bf84bf93c1d089377b2bd29905bc6d/mkdocs.yml#L67-L79